modules/lib/lib-project/src/test/resources/test/GetProjectHandlerTest.js

```diff
@@ -0,0 +1,86 @@
+var assert = require('/lib/xp/testing.js');
+var project = require('/lib/xp/project.js');
+
+var getProjectExpected = {
+    name: "myproject",
+    displayName: "project display name",
+    description: "project description",
+    permissions: {
+        owner: [
+            "user:system:owner2",
+            "user:system:owner1"
+        ],
+        viewer: [
+            "user:system:viewer1"
+        ]
+    },
+    readAccess: {
+        "public": true
+    }
+};
+
+exports.getProject = function () {
+    createProject({owner: ['user:system:owner2', 'user:system:owner1'], viewer: ['user:system:viewer1']});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['createProject' was used before it was defined.](https://app.codacy.com/manual/enonic/xp/pullRequest?prid=5502328)